client/lib/signup/private-by-default.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:signup:private-by-default' );
+
+interface PrivateByDefaultSiteSettings {
+	flowName?: string;
+}
+
+/**
+ * Should the site be private by default
+ * @param siteSettings PrivateByDefaultSiteSettings
+ * @returns `true` for private by default & `false` for not
+ */
+export function shouldBePrivateByDefault( {
+	flowName = '',
+}: PrivateByDefaultSiteSettings ): boolean {
+	if ( flowName.match( /^ecommerce/ ) ) {
+		// ecommerce plans go atomic after checkout. These sites should default to public for now.
+		return false;
+	}
+
+	return true;
+}
+
+/**
+ * Get the numeric value that should be provided to the "new site" endpoint
+ *
+ * @param dependencies The `dependencies` passed to the `apiRequestFunction` step action function call
```

Comment:
Is there a reason this param gets passed to `getNewSitePublicSetting` but not used? Room for future changes?

Reply:
Some pieces of data could be there and not in the stepData

It was just to make it easier if anyone wants to add an exception -- I will pull it out before this lands to save the bytes.